### DISCUSSION TITLE
Connect to a VM controlled by RDP

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5938] [Issues]
Hi,
How can I connect to an instance which is controlled only by RDP?
I've installed the Mesh on a VM in which Ubuntu is installed and I can connect to it only by RDP.
I have only the built in Desktop thingy in the Mesh and besides seeing the instance's desktop everything else works.
When I try to connect I see the message : KVM child process had unexpectedly exited.
Thanks.

**Comment by si458** [comment]
Use the web-rdp option

**Comment by ghost** [comment]
I don't have this option only the built in Desktop

**Reply by si458** [reply]
You need to set `mstsc: true` under ur domain in config.json to enable the web-rdp, and the option will appear next to web-vnc

**Reply by ghost** [reply]
Sorry I didn't see that there is already Web RDP option in the general options.

**Reply by si458** [reply]
Yes the web-rdp should show for all devices where ad the rdp connect in the desktop tab will only show for Windows devices

**Reply by ghost** [reply]
I meant that the web RDP exists also without putting the command in the Config file.

**Reply by si458** [reply]
No, I think u still need the `mstsc: true`

**Reply by ghost** [reply]
You mean that I need the mstsc command to enable it?

**Reply by ghost** [reply]
Is there a way to connect via Desktop connecttion?

**Reply by si458** [reply]
for linux just click connect (make sure you switch from wayland to x11),
but for RDP connect, not for linux that am aware of, you would need to use the Web-RDP option :)
